canvas_modules/common-canvas/src/common-properties/controls/textfield/textfield.jsx

```diff
@@ -108,32 +110,31 @@ class TextfieldControl extends React.Component {
 					labelText={this.props.controlItem}
 					hideLabel={this.props.tableControl}
 					aria-label={this.props.control.labelVisible ? null : this.props.control?.label?.text}
-					ref={(ref) => (this.textInputRef = ref)}
+					ref={this.textInputRef}
 					readOnly={this.props.readOnly}
 				/>
 			);
-		}
 
-		let display = textInput;
-		if (this.props.tableControl) {
-			const tooltipProps = {};
-			if (this.textInputRef) {
-				tooltipProps.truncatedRef = this.textInputRef;
+			if (this.props.tableControl) {
+				const tooltipProps = {
+					truncatedRef: this.textInputRef
```

Comment:
Table cells are being rendered after the table is created. So when this gets passed into the table as data object, the ref is undefined. Its only when the input gets modified, and the textfield rerenders, will the ref becomes available and gets passed into the TruncatedContentTooltip component. 

This new way is to create a ref and pass that pointer to the TruncatedContentTooltip. Once current becomes available, that will then be used for the length comparison tooltip.jsx.